src/ui/PirateRadioModal.jsx

```
@@ -0,0 +1,99 @@
+        exit={{ scale: 0.95, opacity: 0 }}
+        className={`bg-void-black border-2 border-toxic-green p-6 max-w-lg w-full shadow-[0_0_30px_var(--color-toxic-green-20)] flex flex-col gap-6 ${contentClassName}`}
+      >
+        <div className='flex justify-between items-start border-b-2 border-toxic-green/50 pb-4'>
+          <h2 className='text-3xl font-[Metal_Mania] text-toxic-green glitch-text' data-text={t('ui:pirate_radio.title', { defaultValue: 'PIRATE RADIO BROADCAST' })}>
+            {t('ui:pirate_radio.title', { defaultValue: 'PIRATE RADIO BROADCAST' })}
+          </h2>
+          <button
+            onClick={onClose}
+            className='text-toxic-green hover:text-white hover:bg-toxic-green px-2 border border-transparent hover:border-toxic-green transition-colors font-mono'
+            title={t('ui:button.close', { defaultValue: 'Close' })}
+          >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)